apps/server/src/app/app.ts

```diff
@@ -44,7 +43,6 @@ import {
     toolsRouter,
     publicRouter,
     e2eRouter,
-    adminRouter,
```

Comment:
This should not be removed. It was just added back for queue service debugging

Reply:
Yes, I noticed it but I thought I was a few commits behind. I'll add everything back.

Reply:
Thanks! Gotta love merge conflicts 😄